spec/models/purl_fetcher/record_deletes_spec.rb

```diff
@@ -13,8 +13,8 @@
         described_class.new(druid: 'abc123')
       end
       it 'enqueue a delete job' do
-        expect(Delayed::Job).to receive(:enqueue).with(DiscoveryDispatcher::IndexingJob.new('delete', 'abc123', 'searchworkspreview'))
-        expect(Delayed::Job).to receive(:enqueue).with(DiscoveryDispatcher::IndexingJob.new('delete', 'abc123', 'searchworks'))
+        expect(Delayed::Job).to receive(:enqueue).with(DiscoveryDispatcher::DeleteFromAllIndexesJob.new('delete', 'abc123', 'searchworkspreview'))
+        expect(Delayed::Job).to receive(:enqueue).with(DiscoveryDispatcher::DeleteFromAllIndexesJob.new('delete', 'abc123', 'searchworks'))
         subject.enqueue
```

Comment:
I think that this information is still needed. Unless we modify the `IndexingJob` to take a URL param, we still need to know how to lookup which indexer to send the request to. It definately is confusing, but the "delete from all" really happens in the `RecordDeletes#enqueue` method.